### ISSUE TITLE
Problem w/ leaky_relu

### ISSUE THREAD

**Issue by klawal** [tensorflow/tensorflow#17569]
### Describe the problem
Having problems using `tf.nn.leaky_relu`.  Attempted to deal with the problem by trying out `tf.nn.elu` and `tf.nn.relu`, but both functions resulted in a fairly significant drop in accuracy.  I was originally building the model in Keras, but switched to TensorFlow after having a problem with the corresponding leaky_relu function in Keras.  The TensorFlow and Keras source code and error logs are below.  Thank you in advance!

### Source code / logs
TensorFlow Source Code
```
def conv2d_w_leakyrelu(x, W, b, s = 1, name = "conlayer_i"): 
    x = tf.nn.conv2d(input = x, filter = W, strides = [1, s, s, 1], data_format = "NHWC", padding = "SAME", name = name)
    x = tf.nn.bias_add(value = x, bias = b)
    # return tf.nn.elu(features = x) # TENSORFLOW PROBLEMS WITH LEAKY_RELU, SO USING ELU ACTIVATION FOR NOW
    return tf.nn.leaky_relu(features = x, alpha = lr_alpha)
```

Keras Source Code
```
model.add(Conv2D(32, kernel_size = (im_h, im_w), strides = (s, s), padding = "same", kernel_initializer = KI, input_shape = input_shape))
    model.add(LeakyReLU(alpha = leakiness))
```

Below is the TensorFlow error message
<img width="962" alt="screen shot 2018-03-08 at 3 13 12 pm" src="https://user-images.githubusercontent.com/10000384/37174004-489b7656-22e3-11e8-9118-060eba773965.png">

Below is part of the Keras error message
<img width="850" alt="screen shot 2018-03-08 at 3 23 54 pm" src="https://user-images.githubusercontent.com/10000384/37174559-df014b2e-22e4-11e8-961b-3be3f247cd6a.png">

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by klawal** [comment]
**Custom Code? **: No 
**TensorFlow Version: ** 1.3.0
**CUDA/cuDNN Version:** V8.0.61
**GPU model and memory**: Tesla K80
**Exact Command to Reproduce**: Call the functions

**Comment by ppwwyyxx** [comment]
Your TF version is too old.

**Comment by klawal** [comment]
@ppwwyyxx Thank you.
